bucket/ricochet.json

```diff
@@ -0,0 +1,24 @@
+{
+    "version": "1.1.4",
+    "description": "Anonymous peer-to-peer instant messaging.",
+    "homepage": "https://ricochet.im",
+    "license": "BSD-3-Clause",
+    "url": "https://github.com/ricochet-im/ricochet/releases/download/v1.1.4/ricochet-1.1.4-win-install.exe#/dl.7z",
```

Comment:
Why this rename, when you are using `innosetup`??

Reply:
Sorry, my fault...